task/04-arrays-tasks.js

```diff
@@ -536,7 +608,8 @@ function selectMany(arr, childrenSelector) {
  *   [[[ 1, 2, 3]]], [ 0, 0, 1 ]      => 2        (arr[0][0][1])
  */
 function getElementByIndexes(arr, indexes) {
-    throw new Error('Not implemented');
+   const strOfIndex = indexes.reduce((acc, item) => acc + `[${item}]`, '');
+   return eval(`arr${strOfIndex}`);
```

Comment:
There's no reason to use `eval`, see pls https://javascript.info/eval, you may solve this with https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce